package-managers/nuget.md

```diff
@@ -42,15 +42,23 @@ We will need the NuGet V3 URL, so let's copy it for later use.
 
 ### Visual Studio
 
-#### Public feed
+#### Private feed
 
 We will need the URL for the feed we want to connect to. Learn more about [determining the feed URL](#determining-the-feed-url).
 
-#### Private feed
+We can register a MyGet feed the same way we register any NuGet package source by using the Package Manager Settings dialog. We can find it under _Tools > Library Package Manager > Package Manager Settings_ in the Visual Studio menu.

```

Comment:
Minor nit: instead of `>` for separating menu entries, use `|`. Ideal style:

**_Tools | Something | Something else_**